connector/__manifest__.py

```diff
@@ -1,24 +1,21 @@
-# -*- coding: utf-8 -*-
 # Copyright 2013-2017 Camptocamp SA
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html)
 
-{'name': 'Connector',
- 'version': '12.0.1.1.0',
- 'author': 'Camptocamp,Openerp Connector Core Editors,'
-           'Odoo Community Association (OCA)',
- 'website': 'http://odoo-connector.com',
- 'license': 'AGPL-3',
- 'category': 'Generic Modules',
- 'depends': ['mail',
-             'queue_job',
-             'component',
-             'component_event',
-             ],
- 'data': ['security/connector_security.xml',
-          'security/ir.model.access.csv',
-          'views/checkpoint_views.xml',
-          'views/connector_menu.xml',
-          'views/res_partner_views.xml',
-          ],
- 'installable': False,
- }
+{
+    "name": "Connector",
+    "version": "13.0.1.0.0",
+    "author": "Camptocamp,Openerp Connector Core Editors,"
+    "Odoo Community Association (OCA)",
+    "website": "http://odoo-connector.com",
+    "license": "LGPL-3",
```

Comment:
So ... `connector` is LGPL-3 while `queue_job` (dependency) is AGPL-3? Is this legally possible?

Reply:
You have a point. The legal question is hard to answer.
In any case, I think we should change `queue_job`'s license to LGPL-3 for the same reason as mentioned on #356

Reply:
Here's the PR for changing the license https://github.com/OCA/queue/pull/184

Reply:
Here's the PR for changing the license https://github.com/OCA/queue/pull/184